.github/workflows/py_test.yml

```diff
@@ -26,11 +26,11 @@ jobs:
 
       - name: Analysing the code with pylint
         run: |
-          pylint \
-            --fail-under=6.0 \
-            --ignore-patterns=test_.*?py \
-            --max-line-length=180 \
-            $(git ls-files '*.py')
+          pylint $(git ls-files '*.py')
```

Comment:
1. minimal pylint score is set in .pylintrc as` fail-under=6` 
2. what is wrong with both failing? With this implementation I can setup only once

Reply:
`fail-under `is set to 8.5, it will be changed to 9.5 in #31 
pylint and black is now parallel
parallel running column2vec test will be solved in #32